src/libostree/libostree-devel.sym

```diff
@@ -20,7 +20,11 @@
    - uncomment the include in Makefile-libostree.am
 */
 
-
+LIBOSTREE_2022.4 {
+global:
+  ostree_fd_get_all_xattrs;
+  ostree_get_all_xattrs_at;
```

Comment:
Naming related, could we try to reduce the distance between these two via a common prefix like `ostree_xattrs_get_all_{at,from_fs}` or `ostree_fsmeta_*`?

Reply:
Good suggestion, fixed that!